harvester_e2e_tests/scenarios/test_create_vm.py

```diff
@@ -714,9 +714,9 @@ def _check_vm_instance_migrated():
                 _check_vm_instance_migrated,
                 step=5,
                 timeout=request.config.getoption('--wait-timeout'))
-        except polling2.TimeoutException as e:
+        except polling2.TimeoutException:
             errfmt = 'Timed out while waiting for VM to be migrated: %s'
-            raise AssertionError(errfmt % vm_json['metadata']['name']) from e
+            raise AssertionError(errfmt % vm_json['metadata']['name'])
```

Comment:
We don't necessarily need to change this :smile: - this is more of a general comment:

I just noticed on the dict of `vm_json` we may also experience a `KeyError` while trying to build the `AssertionError` message.  Like the few other comments below.